comet/core/config_validation.py

```
@@ -62,9 +62,14 @@ def _parse_and_validate_config(b64config: str):
         validated_config = ConfigModel(**config)
         validated_config = validated_config.model_dump()
 
-        options = validated_config["options"]
+        default_options = rtn_settings_default_dumped["options"]
```

Ruff (0.15.21)

[error] Undefined name `rtn_settings_default_dumped`

(F821)